packages/blueprint/package.json

```
@@ -4,18 +4,18 @@
   "version": "1.0.2",
```

```suggestion
  "version": "1.0.3",
```
Should we bump versions on changes to dependencies?

---

depends, if the changed dependency effect the package so much is validates bumping its version I'd say yes :)

---

But if you look closely, no dependencies has actually changed, only reordered.
yarn...smh